server/controllers/admin.py

```diff
@@ -637,9 +637,21 @@ def enrollment(cid):
     return render_template('staff/course/enrollment.html',
                            enrollments=students, staff=staff,
                            single_form=single_form,
+                           unenroll_form=forms.CSRFForm(),
                            courses=courses,
                            current_course=current_course)
 
+@admin.route("/course/<int:cid>/<int:user_id>/unenroll", methods=['POST'])
+@is_staff(course_arg='cid')
+def unenrollment(cid, user_id):
+    user = User.query.filter_by(id=user_id).one_or_none()
+    enrollment = user.is_enrolled(cid);
+    if user and enrollment:
+        Enrollment.unenroll(enrollment)
```

Comment:
This should just be a method instead of a class method. 
`enrollment.unenroll()`